### DISCUSSION TITLE
usage with laravel and esbuild modules like vite

### DISCUSSION THREAD

**Discussion by ghost** [inorganik/countUp.js#308] [Q&A]
can you discuss usage with laravel and esbuild modules like vite?

**Comment by inorganik** [comment]
You're gonna need to give me a bit more context - can you describe how you are loading other libraries? Or anything about your setup?

**Comment by ghost** [comment]
Go it working: 
Larvel 10

app.js
`import * as Popper from "popperjs-app";
window.Popper = Popper;
import "bootstrap-app";
import { CountUp } from 'countup.js';


const countUp = new CountUp('jobsCount', 583, { enableScrollSpy: true });
`

package.json
`    "dependencies": {
        "bootstrap-app": "npm:bootstrap@^5.3.0-alpha1",
        "countup": "npm:countup.js@^2.6.0",
        "popperjs-app": "npm:@popperjs/core@^2.11.6"
    }
}
`
